sdk/core/token_library/token_unseal.S

```diff
@@ -80,18 +80,18 @@ __sealingkey_dynamic:
 
   /*
    * Load software type tag.  This will not trap, thanks to above tag check and
-   * because IRQs are deferred (see our export entry below)
+   * because IRQs are deferred (see our export entry below), unless the
```

Comment:
Do we actually need interrupts disabled here?

Reply:
Oh, indeed, yes, if this load fails due to a clear tag then we have no secrets to leak.  Great.

I'd like to offer some limited unwind functionality in privileged libraries in general.  Bonus points if we can make it sufficiently expressive to implement the current "trap in cross-call is forces unwind of trusted stack" policy. :)